explorer/client/cypress/e2e/e2e.cy.ts

```diff
@@ -0,0 +1,284 @@
+        });
+
+        it('can be reached through URL', async () => {
+            await page.goto(`${BASE_URL}/transactions/${successID}`);
+            const value = await cssInteract(page)
+                .with('#transactionID')
+                .get.textContent();
+            expect(value.trim()).toBe(successID);
+        });
+        it('correctly renders days and hours', async () => {
+            await page.goto(`${BASE_URL}/transactions/${successID}`);
+            const value = await cssInteract(page)
+                .with('#timestamp')
+                .get.textContent();
+            expect(value.trim()).toBe(
+                '17 days 1 hour ago (15 Dec 2024 00:00:00 UTC)'
+            );
+        });
+        it('correctly renders a time on the cusp of a year', async () => {
+            const otherID = 'GHTP9gcFmF5KTspnz3KxXjvSH8Bx0jv68KFhdqfpdK8=';
+            await page.goto(`${BASE_URL}/transactions/${otherID}`);
+            const value = await cssInteract(page)
+                .with('#timestamp')
+                .get.textContent();
+            expect(value.trim()).toBe(
+                '1 min 3 secs ago (01 Jan 2025 01:12:07 UTC)'
+            );
+        });
+        it('correctly renders a time diff of less than 1 sec', async () => {
+            const otherID = 'XHTP9gcFmF5KTspnz3KxXjvSH8Bx0jv68KFhdqfpdK8=';
+            await page.goto(`${BASE_URL}/transactions/${otherID}`);
+            const value = await cssInteract(page)
+                .with('#timestamp')
+                .get.textContent();
+            expect(value.trim()).toBe('< 1 sec ago (01 Jan 2025 01:13:09 UTC)');
+        });
+    });*/
+
+    describe('Owned Objects have links that enable', () => {
+        it('going from address to object and back', () => {
+            cy.visit('/addresses/receiverAddress');
+            cy.get(nftObject(1)).click();
+            cy.get('#objectID').contains('player1');
+            cy.get(ownerButton).click();
+            cy.get('#addressID').contains('receiverAddress');
+        });
+
+        it('going from object to child object and back', () => {
+            cy.visit('/objects/player2');
+            cy.get(nftObject(1)).click();
+            cy.get('#objectID').contains('Image1');
+            cy.get(ownerButton).click();
+            cy.get('#objectID').contains('player2');
+        });
+
+        it('going from parent to broken image object and back', () => {
+            const parentValue = 'ObjectWBrokenChild';
+            cy.visit(`/objects/${parentValue}`);
+            cy.get(nftObject(1)).click();
+            cy.get('#noImage');
+            cy.get(ownerButton).click();
+            cy.get('#loadedImage');
+        });
+    });
+
+    describe('PaginationWrapper has buttons', () => {
+        it('to go to the next page', () => {
+            const address = 'ownsAllAddress';
+            cy.visit(`/addresses/${address}`);
+            cy.get('[data-testid="nextBtn"]').filter(':visible').click();
```

Comment:
Why has `filter(':visible')` been added? These buttons are always visible under the new design.

Reply:
There are two versions of these buttons rendered at the same time, one for mobile, one for desktop. This filter just grabs the currently visible ones. I was hoping to take a swing at refactoring the buttons at a later point to avoid the filter here.

Reply:
I understand. When the website is in Desktop, the buttons for mobile are not 'invisible' but rather 'hidden'. As a result, the addition of `filter(':visible')` is not required because these buttons are not in the DOM.

Reply:
This isn't really true, the buttons are still in the DOM but are hidden by media queries.

Reply:
* By default Cypress runs tests in a viewport that has a width of 1000px and height of 660px. As we use an `sm` cut-off to switch between mobile and desktop, this means that we switch to mobile view when width is below 640px and so the Cypress tests are being run in Desktop mode. This means that the mobile buttons are `hidden` --> please see `src/components/pagination/Pagination.module.css`.
* `hidden` and `invisible` are two distinct Tailwind concepts that reflect two distinct CSS concepts. Please consult the following URLs in the Tailwind documentation: https://tailwindcss.com/docs/visibility and https://tailwindcss.com/docs/display#hidden 
* Based on this, `filter(':visible')` should be removed.

Reply:
I see - rather than assigning unique data-testid values for Desktop and Mobile buttons, which would render `filter(':visible')` unnecessary, we're having to use jQuery to differentiate between the two. This will do for now.

Reply:
Thank you for clarifying this on Slack.